modules/jaggery-apps/portal/gadgets/consent_management/js/gadget.js

```diff
@@ -385,18 +386,18 @@ function addActions(container) {
     });
 }
 
-function showWarning() {
+function showWarning(container) {
```

Comment:
what's the usage of container here?

Reply:
container is an element need to be passed in the callBack function of the warning/confirmation dialog

Reply:
But the callback function is empty here. It's used in showConfirm()

Reply:
Fixed